Frameworks/Foundation/NSKeyValueCoding.mm

```diff
@@ -256,9 +256,9 @@ static bool tryGetArrayAdapter(id self, const char* key, id* ret) {
 }
 
 static bool tryGetSetAdapter(id self, const char* key, id* ret) {
-    auto countSelectorString(woc::string::format("countOf%c%s", toupper(key[0]), key + 1));
```

Comment:
you can use the `append(size_t, CharT)` overload here instead of constructing a temporary string